src/Uno.UI/UI/Xaml/Controls/Pivot/Pivot.cs

```diff
@@ -255,5 +255,22 @@ private static void OnTitlePropertyChanged(DependencyObject dependencyObject, De
 				}
 			}
 		}
+
+		private void OnSelectedItemPropertyChanged(object oldValue, object newValue)
+		{
+			var removedItems = oldValue == null ? new object[0] : new [] { oldValue };
+			var addedItems = new [] { newValue };
```

Comment:
```suggestion
			var addedItems = newValue == null ? new object[0] : new [] { newValue };
```

the `newValue` might be `null` (manual set of the `SelectedItem` or removing / clearing the `ItemsSource`).